src/qgitrepository.h

```diff
@@ -681,6 +667,31 @@ namespace LibQGit2
              */
             bool shouldIgnore(const QString &path) const;
 
+            /**
+             * Set the identity to be used for writing reflogs.
+             *
+             * @param name The name to use for the reflog entries
+             * @param email The email to use for the reflog entries
+             *
+             * @throws LibQGit2::Exception
+             */
+            void setIdent(const QString &name, const QString &email);
```

Comment:
For consistency, this function should take an `Ident` argument.

Reply:
It now takes an `Identity` argument.